src/main/shell-session/shell-session.ts

```diff
@@ -158,6 +158,8 @@ export abstract class ShellSession {
         cwd,
         env,
         name: "xterm-256color",
+        // TODO: Something else is broken here so we need to force the use of winPty on windows
+        useConpty: false,
```

Comment:
what does this do on mac and linux?

Reply:
Nothing, this option is ignored on non-windows OS's

Reply:
mention or reference the issue this is a workaround for, in the comment?